index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="header">
+      <div class="nav">
+        <div class="nav__left">
+          <h3 class="nav__logo">Medical Core</h3>
+          <div class="nav__items">
+            <a class="nav__item" href="#">Home</a>
+            <a class="nav__item" href="#">Product</a>
+            <a class="nav__item" href="#">Pricing</a>
+            <a class="nav__item" href="#">Contact</a>
+          </div>
+        </div>
+        <div class="nav__right">
+          <button class="btn btn--link">Login</button>
+          <button class="btn">Join Us <img src="img/Vector.svg" /></button>
+        </div>
+      </div>
+      <div class="content">
+        <h5 class="content__header">Welcome</h5>
+        <h1 class="content__title">A Great Place To Receive Care</h1>
+        <p class="content__description">Get you best loking smile now!</p>
+        <div>
```

Comment:
DIV elements with no class. These are unnecessary. If you still need it for some reasons give it a class.